sympy/physics/control/lti.py

```diff
@@ -0,0 +1,553 @@
+        >>> G1.num
+        p*s + s**2 + 3
+        >>> G2 = TransferFunction((p + 5)*(p - 3), (p - 3)*(p + 1), p)
+        >>> G2.num
+        (p - 3)*(p + 5)
+
+        """
+        return self._num
+
+    @property
+    def den(self):
+        """Returns the Denominator of the Transfer Function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s + 4, p**3 - 2*p + 4, s)
+        >>> G1.den
+        p**3 - 2*p + 4
+        >>> G2 = TransferFunction(3, 4, s)
+        >>> G2.den
+        4
+
+        """
+        return self._den
+
+    @property
+    def var(self):
+        """Returns the variable that is used to anchor the symbols.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(p**2 + 2*p + 4, p - 6, p)
+        >>> G1.var
+        p
+        >>> G2 = TransferFunction(0, s - 5, s)
+        >>> G2.var
+        s
+
+        """
+        return self._var
+
+    @property
+    def bound_symbols(self):
+        """Returns only variables that are dummy variables.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, l
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s + l, s - 2*p, s)
+        >>> G1.bound_symbols
+        [s]
+        >>> G2 = TransferFunction(p**2 + 2*p + 4, p - 6, p)
+        >>> G2.bound_symbols
+        [p]
+
+        """
+        return [self.var]
+
+    def _eval_subs(self, old, new):
+        arg_num = self.args[0].subs(old, new)
+        arg_den = self.args[1].subs(old, new)
+        argnew = TransferFunction(arg_num, arg_den, self.var)
+        return self if old in self.bound_symbols else argnew
+
+    def _eval_evalf(self, prec):
+        return TransferFunction(
+            self.num._eval_evalf(prec),
+            self.den._eval_evalf(prec),
+            self.var)
+
+    def _eval_simplify(self, **kwargs):
+        tf = cancel(Mul(self.num, 1/self.den)).as_numer_denom()
+        num_, den_ = tf[0], tf[1]
+        return TransferFunction(num_, den_, self.var)
+
+    def expand(self):
+        """
+        Returns the Transfer Function with Numerator and Denominator
+        in expanded form.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction((a - s)**2, (s**2 + a)**2, s)
+        >>> G1.expand()
+        TransferFunction(a**2 - 2*a*s + s**2, a**2 + 2*a*s**2 + s**4, s)
+        >>> G2 = TransferFunction((p + 3*b)*(p - b), (p - b)*(p + 2*b), p)
+        >>> G2.expand()
+        TransferFunction(-3*b**2 + 2*b*p + p**2, -2*b**2 + b*p + p**2, p)
+
+        """
+        return TransferFunction(expand(self.num), expand(self.den), self.var)
```

Comment:
Why are you not using any hints to expand here? For example, should we be using the 'multinomial` hint? If you know these are always polynomials in `var` then it seems like we should be doing a more specific expansion only with respect to `var`. You don't want other things to expand. If the coefficients to `var` are complex expressions, you wouldn't want to expand those.